pkg/aggregator/aggregator.go

```diff
@@ -122,6 +154,14 @@ var (
 		nil, "Count of hostname update")
 	tlmDogstatsdContexts = telemetry.NewGauge("aggregator", "dogstatsd_contexts",
 		nil, "Count the number of dogstatsd contexts in the aggregator")
+	tlmHugeSeries = telemetry.NewCounter("aggregator", "huge_series",
+		nil, "Count of timeseries with over 100 tags")
```

Comment:
I'd put the limit in the the metric name to be more descriptive, and make it pick up the constant value, i.e.:

```suggestion
	tlmHugeSeries = telemetry.NewCounter("aggregator", fmt.Sprintf("series_tags_above_%d", hugeTagsetThreshold),
		nil, fmt.Sprintf("Count of timeseries with over %d tags", hugeTagsetThreshold))
```